test/unit/aws/conftest.py

```diff
@@ -1,5 +1 @@
-from test.aws.conftest import (
-    default_asset_id,
-    ec2_cloudformation_yml,
-    test_dummy_ami_id,
-)
+pytest_plugins = "test.aws.fixtures"
```

Comment:
pytest_plugins is deprecated for not-root conftest
https://docs.pytest.org/en/7.1.x/how-to/writing_plugins.html#requiring-loading-plugins-in-a-test-module-or-conftest-file

Reply:
yes, I read it first and decided to ignore it, but the failed ci build seems to enforce the deprecation. 
Please see latest push.